doc/source/whatsnew/v1.2.0.rst

```diff
@@ -291,6 +291,7 @@ Other enhancements
 - Improve numerical stability for :meth:`.Rolling.skew`, :meth:`.Rolling.kurt`, :meth:`Expanding.skew` and :meth:`Expanding.kurt` through implementation of Kahan summation (:issue:`6929`)
 - Improved error reporting for subsetting columns of a :class:`.DataFrameGroupBy` with ``axis=1`` (:issue:`37725`)
 - Implement method ``cross`` for :meth:`DataFrame.merge` and :meth:`DataFrame.join` (:issue:`5401`)
+- :func:`read_csv/sas/json` return a context manager when called with ``chuncksize``/``iterator`` (:issue:`38225`)
```

Comment:
can you change this slightly, that with turn a reader which can be used in a context manager